### PR TITLE
minor edits, added content from issue 937

### PR DESCRIPTION
**Pull request by Ampersandstorm**
This PR adds mention of support panel logs include upower output in last section.

### CODE REVIEW

File: content/battery.md [hunk 1]
```diff
@@ -68,6 +70,10 @@ man tlp-stat
 
 ---
 
+### Powertop
+
+Powertop monitors running processes and applications to determine how much power is being consumed. Powertop can also use application battery usage data to create custom power profiles. We recommend installing and running TLC before using Powertop.
```
Comment (jacobgkau) [inline content/battery.md:75]: > We recommend installing and running TLC

Is this supposed to be TLD?

Comment (Ampersandstorm) [reply content/battery.md]: Yes, good catch.

**Comment by jacobgkau** [comment]
In your commit messages, it's better to include a brief description of what the commit is actually doing/changing (e.g. "mention support panel logs include upower output") instead of just referring to issue numbers. Issue numbers are GitHub-specific metadata.

File: content/battery.md [hunk 2]
```diff
@@ -160,7 +168,9 @@ sudo ifconfig enp4s0f2 up
 upower -d
 ```
 
-This will show the information that your computer can read about the battery.
+This will show the information that your computer can read about the battery. The output from `upower -d` is also included when generating log files in Settings -> Support -> Create Log Files:
+
+![Create Log Files](/images/battery/create-logs.png)
```
Comment (jacobgkau) [inline content/battery.md:175]: The window is unfocused in this screenshot, so the button looks greyed out. Can we get a screenshot with the window focused?

**Review by jacobgkau — APPROVED** [review]
All of these changes increase consistency within the article. The new content regarding support-panel is accurate, and the new content regarding powertop is consistent with what's already in the article.

**Review by ahoneybun — APPROVED** [review]
Changes look good and help clear up the article more.

**Pull request merged**